virtcontainers/qemu_amd64.go

```diff
@@ -129,6 +129,7 @@ func (q *qemuAmd64) bridges(number uint32) {
 
 func (q *qemuAmd64) cpuModel() string {
 	cpuModel := defaultCPUModel
+	cpuModel += ",kvm=off"
```

Comment:
If this is really the solution for Nvidia GPUs, couldn't we just document to add `kvm=off` to `cpu_features=` in `configuration.toml`?

Aside: If the number of GPU-specific settings in `configuration.toml` grows, we might want to think about adding an actual GPU config option which if set automatically sets these "ancillary" config options to make life easier for GPU users.

Reply:
:+1:  please don't hard code this , instead you can document it as a GPU-specific setting for some specific GPUs